src/arrayMethodJoin.js

```diff
@@ -5,7 +5,25 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let finalSeparator = separator;
+
+    if (finalSeparator === undefined) {
+      finalSeparator = ',';
+    }
```

Comment:
It would be better to set a default value, instead of this check. For example function(separator = ', ')